stix2/markings/__init__.py

```diff
@@ -212,3 +212,16 @@ def is_marked(obj, marking=None, selectors=None, inherited=False, descendants=Fa
         result = result or object_markings.is_marked(obj, object_marks)
 
     return result
+
+
+class MarkingsMixin():
+    pass
+
+
+# Note that all of these methods will return a new object because of immutability
```

Comment:
Oooo, this is super-slick. Because those functions already take `obj`as their first argument, they can become methods on the Mixed-in objects (where `obj` is implicitly `self`).